package.json

```diff
@@ -27,12 +27,14 @@
     "fqm-execution": "^0.8.0",
     "lodash": "^4.17.21",
     "mongodb": "^4.1.0",
+    "save-dev": "0.0.1-security",
```

Comment:
Wow... this is a package that used to be a malicious thing since it could be created by a typo. Definitely remove it.